README.md

```diff
@@ -132,15 +124,15 @@ The parameters are pretty clear the only thing you need to care is parameters_li
     (Unsafe!) Do regular transaction to a contract, you need pass the private key of sender and address of contract  
     you need to initialize a project before using this command.
     
-    bond|bnd [options] <priv_key> <address> <amount> <fee> <public_key>  
+    bond|bnd [options] <public_key> <address> <amount> <fee>  <priv_key>
```

Comment:
In bind you don't need address, public key is required. I think it is a little bit confusing. Please feel free to discuss about it.

Reply:
the bond_tx params is not accurate!

Reply:
Yes I agree, the implementation is done without address . I will update the readme.